src/ibek/ioc.py

```diff
@@ -54,60 +55,69 @@ class Entity(BaseSettings):
     entity_enabled: bool = Field(
         description="enable or disable this entity instance", default=True
     )
-    __definition__: EntityDefinition
+    __definition__: EntityModel
```

Comment:
This should probably have a new name too

Reply:
I'm not getting you - that is the new name we settled on